rust/osm2lanes/src/test.rs

```diff
@@ -377,6 +377,10 @@ mod tests {
                 let expected_road = test.expected_road();
                 match road_from_tags {
                     Ok(road_from_tags) => {
+                        if test.test_has_warnings() && road_from_tags.warnings.is_empty() {
+                            test.print();
+                            println!("Expected warnings. Try removing `ignore_warnings`.");
```

Comment:
Should this be a panic / error? If the test passes in Rust, STDOUT isn't shown at all